plugins/woocommerce/src/Internal/Admin/Orders/Edit.php

```diff
@@ -0,0 +1,202 @@
+		$current_screen->is_block_editor( false );
+		$this->screen_id = $current_screen->id;
+		$this->add_order_meta_boxes( $this->screen_id, __( 'Order', 'woocommerce' ) );
+		$this->add_order_specific_meta_box();
+
+		/**
+		 * From wp-admin/includes/meta-boxes.php.
+		 *
+		 * Fires after all built-in meta boxes have been added. Custom metaboxes may be enqueued here.
+		 *
+		 * @since 3.8.0.
+		 */
+		do_action( 'add_meta_boxes', wc_get_page_screen_id( 'shop-order' ), $this->order );
+		$this->enqueue_scripts();
+	}
+
+	/**
+	 * Setup order object to be used in metaboxes.
+	 *
+	 * @return void
+	 */
+	private function initialize_order_object() {
+		global $theorder;
+		$action = sanitize_text_field( wp_unslash( isset( $_GET['action'] ) ? $_GET['action'] : '' ) );
+		switch ( $action ) {
+			case 'edit':
+				$this->order = wc_get_order( absint( isset( $_GET['id'] ) ? $_GET['id'] : 0 ) );
+				$this->verify_edit_object_and_permission();
+				break;
+			case 'new':
+				$this->order = new \WC_Order();
+				if ( ! current_user_can( 'publish_shop_orders' ) && ! current_user_can( 'manage_woocommerce' ) ) {
+					wp_die( esc_html__( 'You don\'t have permission to create a new order', 'woocommerce' ) );
+				}
+				break;
+			default:
+				wp_redirect( admin_url( 'admin.php?page=wc-orders' ) );
+				exit;
+		}
+		$theorder = $this->order;
+	}
+
+	/**
+	 * Verify that the order object is valid and user has permission to edit it.
+	 */
+	private function verify_edit_object_and_permission() {
+		if ( ! isset( $this->order ) || ! $this->order ) {
+			wp_die( esc_html__( 'You attempted to edit an item that does not exist. Perhaps it was deleted?', 'woocommerce' ) );
+		}
+
+		if ( ! current_user_can( 'edit_others_shop_orders' ) && ! current_user_can( 'manage_woocommerce' ) ) {
+			wp_die( esc_html__( 'You do not have permission to edit this order', 'woocommerce' ) );
+		}
+	}
+
+	/**
+	 * Hooks meta box for order specific meta.
+	 */
+	private function add_order_specific_meta_box() {
+		add_meta_box( 'postcustom', __( 'Custom Fields', 'woocommerce' ), array( $this, 'render_custom_meta_box' ), $this->screen_id, 'normal' );
+	}
+
+	/**
+	 * Render meta box for order specific meta.
+	 */
+	public function render_custom_meta_box() {
+		if ( ! isset( $this->custom_meta_box ) ) {
+			$this->custom_meta_box = new CustomMetaBox();
+		}
+		$this->custom_meta_box->output( $this->order );
+	}
+
+	/**
+	 * Render order edit page.
+	 */
+	public function display() {
+		$this->setup();
+		$this->render_wrapper_start();
+		$this->render_meta_boxes();
+		$this->render_wrapper_end();
+	}
+
+	/**
+	 * Helper function to render wrapper start.
+	 *
+	 * @param string $notice Notice to display, if any.
+	 * @param string $message Message to display, if any.
+	 */
+	private function render_wrapper_start( $notice = '', $message = '' ) {
+		?>
+		<div class="wrap">
+		<h1 class="wp-heading-inline">
+			<?php echo esc_html( 'Edit Order' ); ?>
```

Comment:
```suggestion
			<?php echo esc_html( 'Edit order' ); ?>
```

For consistency with the existing post-based editor:

<img width="202" alt="edit-order-casing" src="https://user-images.githubusercontent.com/3594411/177429203-54d5f698-07ef-46ad-a9d4-b80314a5b568.png">